test/test_dgmulti_1d.jl

```diff
@@ -0,0 +1,34 @@
+module TestExamplesDGMulti1D
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "dgmulti_1d")
```

Comment:
```suggestion
EXAMPLES_DIR = examples_dir(), "dgmulti_1d")
```

Reply:
I think there was a syntax error in the suggestion. I just did `joinpath(examples_dir(), "dgmulti_1D")`